po/ta.po

```diff
@@ -0,0 +1,575 @@
+# Flatseal- Tamil translation.
+# Copyright (C) 2022 THE Flatseal'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+# K.B.Dharun Krishna (kbdharun) <kbdharunkrishna@gmail.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-17 19:49-0400\n"
+"PO-Revision-Date: 2022-09-03 12:35+0530\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: ta\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "பிளாட்சீல்"
```

Comment:
Please check these `TRANSLATORS: Don't translate this text` messages :pray:

Reply:
Oops, Sorry I will update it.